etc/tinyproxy.conf.in

```diff
@@ -205,6 +205,14 @@ Allow ::1
 # users.
 #BasicAuth user password
 
+# BasicAuthRealm : In case BasicAuth is configured .. the "realm" information.
+# "Proxy Authentication Required" status http 407 "error-response" can be
+# customized.
+#
+# - defaults in code to "Tinyproxy" (PACKAGE_NAME) .. if not configured
+# - limited to 255 characters. Additional characters will be skipped / ignored.
```

Comment:
is this 255 character limit described in the http spec ? or is it just an arbitrary limit you chose for the char buffer ? if the former, i think it would be best to throw an error in the config file parser code if the length is exceeded. if not, there's no reason to restrict the length.

Reply:
I had not found a lenght limitation / specification in the related RFCs i had revied.

But there needs to be a limit. This string is part of a https reponse header ... and many proxies have http header length limits. We faced there multiple issues in the past in the area of to long request header lengt.

A few products have 4KB as limit ... others 8 KB on request and response processing.
Unrestricted setting i would based on this knowledge not allow.

Due the fakt that the "realm" information should in case of basic auth indicate a sope of protection
https://datatracker.ietf.org/doc/html/rfc7235#section-2.2
i have chosen 255 based on the practices i have seen on other project related to http intefaces.

I do not really see a problem with the limit itself, as long the limitation is clearly described that it exists.

If you want some other limit, sure, tell me your wish, which limit you would want to choose ?

Reply:
I have removed it again and rework relevant parts